dev-util/bats/metadata.xml

```diff
@@ -1,18 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>nelo@wallus.de</email>
+		<name>Nelo-Thara Wallus</name>
+		<description>Proxied by Polynomial-C</description>
```

Comment:
Please do not add this `<description/>`. It's of no real use to bug-wranglers, and they only mean that we can't fully automate bug assignments because `<description/>`s need to be read by a human.